README.md

```diff
@@ -145,3 +145,14 @@ in a directory other than a git cloned workspace, run:
 `swagger-codegen generate -i path/to/data-repository-openapi.yaml -l spring -c path/to/config.json`
 
 Then copy the files you want into the source tree
+
+## skaffold
+To render your own local skffold.yaml run the following with your initials
```

Comment:
```suggestion
To render your own local skaffold.yaml run the following with your initials
```

Reply:
This seems to still be in: https://github.com/DataBiosphere/jade-data-repo/blame/develop/README.md#L150